public/app/features/explore/ExploreGraphNGPanel.tsx

```diff
@@ -0,0 +1,93 @@
+import { AbsoluteTimeRange, dateTime, GrafanaTheme, LoadingState, PanelData, TimeZone } from '@grafana/data';
+import { getTemplateSrv } from '@grafana/runtime';
+import { Collapse, Icon, useTheme } from '@grafana/ui';
+import { GraphPanel } from 'app/plugins/panel/graph3/GraphPanel';
+import { css, cx } from 'emotion';
+import React, { useState } from 'react';
+
+const MAX_NUMBER_OF_TIME_SERIES = 20;
+
+const getStyles = (theme: GrafanaTheme) => ({
+  timeSeriesDisclaimer: css`
+    label: time-series-disclaimer;
+    width: 300px;
+    margin: ${theme.spacing.sm} auto;
+    padding: 10px 0;
+    border-radius: ${theme.border.radius.md};
+    text-align: center;
+    background-color: ${theme.colors.bg1};
+  `,
+  disclaimerIcon: css`
+    label: disclaimer-icon;
+    color: ${theme.palette.yellow};
+    margin-right: ${theme.spacing.xs};
+  `,
+  showAllTimeSeries: css`
+    label: show-all-time-series;
+    cursor: pointer;
+    color: ${theme.colors.linkExternal};
+  `,
+});
+
+interface Props {
+  data: PanelData;
+  width: number;
+  absoluteRange: AbsoluteTimeRange;
+  timeZone: TimeZone;
+  onUpdateTimeRange: (absoluteRange: AbsoluteTimeRange) => void;
+}
+
+export function ExploreGraphNGPanel({ width, data, timeZone, absoluteRange, onUpdateTimeRange }: Props) {
+  const [showAllTimeSeries, setShowAllTimeSeries] = useState(false);
+  const theme = useTheme();
+  const style = getStyles(theme);
```

Comment:
You can replace this with [useStyles builtin hook](https://github.com/grafana/grafana/blob/master/contribute/style-guides/styling.md#styling-with-theme)